Sage-Bionetworks/Bridge-FitBit-Scheduler
# initial implementation

Testing done:
* mvn verify (unit tests, FindBugs, Jacoco test coverage)
* manual tests

Note that this doesn't include Travis build and deployment, which is now supported for Lambda. We will probably be replacing this with the Smart Scheduler early in 2018, and we will be adding Travis build to Lambda at that point.
